testdata/project-v3-multimodule/PROJECT

```diff
@@ -0,0 +1,109 @@
+domain: testproject.org
+layout:
+- go.kubebuilder.io/v3
+- multi-module.go.kubebuilder.io/v1-alpha
+multigroup: true
+plugins:
+  multi-module.go.kubebuilder.io/v1-alpha:
+    ApiGoModCreated: true
```

Comment:
We do not need to track the resources for this one IHMO
If we enable that then it should be valid for the whole project